internal/testbed/linux-services/package-tests.sh

```diff
@@ -28,6 +29,13 @@ if [[ ! -f "$PKG_PATH" ]]; then
     exit 1
 fi
 
+# translate distro x86 arch to normal name
```

Comment:
```suggestion
# translate distro x86 arch to podman arch name
```

Reply:
but it is the generally used name i believe 😅

Reply:
Oh, okay, I guess I am not spending enough time with folks who are discussing arch enough, I guess I am just not familiar with it 😅

Reply:
at least goreleaser uses `amd64` as default and we override it 😄